src/client/components/HealthCareApp.jsx

```diff
@@ -74,10 +76,38 @@ class HealthCareApp extends React.Component {
     this.scrollToTop();
   }
 
-  handleSubmit() {
+  handleSubmit(e) {
+    e.preventDefault();
     const path = this.props.location.pathname;
-    this.context.store.dispatch(updateSubmissionStatus('applicationSubmitted'));
-    this.context.store.dispatch(updateCompletedStatus(path));
+    const store = this.context.store;
+    const veteran = store.getState().veteran;
+
+    store.dispatch(updateSubmissionStatus('submitPending'));
+    store.dispatch(updateCompletedStatus(path));
+
+    // Pretty-print the data that we're about to POST
+    console.log(JSON.stringify(veteran, null, 2));
+
+    // POST data to endpoint
+    fetch('/hca/', {
```

Comment:
This is already here! (sorta).  Try `/v1/api/submit`.

However if you look at:

https://github.com/department-of-veterans-affairs/healthcare-application/blob/master/hca-api-stub/voa-rest.js#L50

you'll notice we have the wrong verb. Oops.